src/helpers.ts

```diff
@@ -50,6 +50,8 @@ export function addKeys(
     node: VNode,
     key: string = Math.random().toString()
 ): VNode {
+    if (!node) return;
```

Comment:
I would throw here too

Reply:
It's not meant to be an error check.  I added that line to skip falsy children VNodes like `null` and `undefined`, which is something  `@cycle/dom` offers.